base/install_motor.sh

```diff
@@ -34,3 +36,30 @@ cd ..
 
 make -j${JOBS}
 make clean
+
+cd $EPICS_MODULES_PATH
+
+download_github_module dls-controls pmac $PMAC_VERSION
+
+rm pmac/configure/RELEASE.local.linux-x86_64
+rm pmac/configure/RELEASE.linux-x86_64.Common
+rm pmac/configure/CONFIG_SITE.linux-x86_64.Common
+
+echo "
+BUILD_IOCS=NO
+USE_GRAPHICSMAGICK=NO
+
+SSH             = $(pkg-config --variable=prefix libssh2)
```

Comment:
And consider making this `SSH=YES`

Reply:
I've thought about that too. We should just remember that `SSH=NO` doesn't mean what it looks like =P.